docker/start.py

```diff
@@ -213,6 +214,13 @@ def main(args, environ):
     if "-m" not in args:
         args = ["-m", synapse_worker] + args
 
+    jemallocpath = f"/usr/lib/{platform.machine()}-linux-gnu/libjemalloc.so.2"
```

Comment:
```suggestion
    jemallocpath = "/usr/lib/%s-linux-gnu/libjemalloc.so.2" % (platform.machine(),)
```

I think we are still using these docker files for Py3.5 for CI? May very well be wrong but seems easier just to change it.

Reply:
Good catch.